### PR TITLE
Add logs for mtp_loss

### PR DESCRIPTION
**Pull request by From00**
### PR Category
Distributed Strategy

### PR Types
New features

### Description
添加MTP loss 打印

### 是否引起精度变化
否

### CODE REVIEW

**Review by Waynezee — DISMISSED** [review]

**Review by risemeup1 — APPROVED** [review]
LGTM

File: src/paddlefleet/models/common/language_loss/language_loss.py [hunk 1]
```diff
@@ -514,14 +515,19 @@ def padding(tensor, left=False, pad_len=1):
...
-            # Store detached MTP loss tensors into class-level tracker.
+            # Store detached MTP loss tensors into class-level tracker and global_training_logs.
             # Use .detach() instead of .item() to avoid GPU synchronization on every
             # micro-batch. The trainer will call .item() only at logging steps.
             for i, loss_val in enumerate(mtp_loss):
                 LanguageLoss.mtp_loss_tracker[f"mtp_{i + 1}_loss"] = (
                     loss_val.detach()
                 )
 
+            logs = get_global_training_logs()
```
[the remark this answers is not in the document]
Comment (From00) [reply src/paddlefleet/models/common/language_loss/language_loss.py:526]: 不会同时调用

File: src/paddlefleet/models/common/language_loss/language_loss.py [hunk 2]
```diff
@@ -590,14 +596,20 @@ def forward(self, dict_args: dict | list, labels: Tensor) -> Tensor:
...
+        # Store detached MTP loss tensors into class-level tracker and global_training_logs.
         # Use .detach() instead of .item() to avoid GPU synchronization on every
         # micro-batch. The trainer will call .item() only at logging steps.
         for i, loss_val in enumerate(mtp_loss):
             MainLanguageLoss.mtp_loss_tracker[f"mtp_{i + 1}_loss"] = (
                 loss_val.detach()
             )
 
+        # Also write to global_training_logs for ernie5 trainer to read
+        logs = get_global_training_logs()
```
[the remark this answers is not in the document]
Comment (From00) [reply src/paddlefleet/models/common/language_loss/language_loss.py:608]: 同上，后续统一
